charts/kyverno/values.yaml

```diff
@@ -297,17 +297,21 @@ config:
   - '[*,kube-system,*]'
   - '[*,kube-public,*]'
   - '[*,kube-node-lease,*]'
+  - '[*,kyverno,*]'
```

Comment:
We need to find the Kyverno namespace dynamically, as Kyverno can be installed in a different namespace instead of `kyverno`.